mbuild/tests/test_compound.py

```diff
@@ -16,10 +16,18 @@
                              has_intermol,
                              has_openbabel,
                              has_networkx,
-                             has_py3Dmol,
-                             has_nglview)
+                             has_py3Dmol)
 from mbuild.tests.base_test import BaseTest
 
+
+try:
+    import nglview
+    has_nglview = True
+    del nglview
+except ImportError:
```

Comment:
@jennyfothergill. I am using a linux machine and I cannot replicate the `Error` described in #840. Even though moving this tests makes sense, I think it might be nice to catch and silence the particular error thrown.

Reply:
I think the error comes from not having installed `jupyter labextension install nglview-js-widgets`. Also this error is in Jupyter Lab, not notebook. It may even be the combination of having nglview installed without the labextension.